Fix flake8 errors in test_l7policy_adapter test file

#### Description
Four flake8 failures seen in this test file:

```
./f5_openstack_agent/tests/functional/adapters/test_l7policy_adapter.py:108:42: F999 dictionary key 'name' repeated with different values
./f5_openstack_agent/tests/functional/adapters/test_l7policy_adapter.py:109:25: F999 dictionary key 'name' repeated with different values
./f5_openstack_agent/tests/functional/adapters/test_l7policy_adapter.py:415:42: F999 dictionary key 'name' repeated with different values
./f5_openstack_agent/tests/functional/adapters/test_l7policy_adapter.py:416:25: F999 dictionary key 'name' repeated with different values
ERROR: InvocationError: '/root/devenv/f5-openstack-agent/.tox/style/bin/flake8 ./f5_openstack_agent'
```

--- a/f5_openstack_agent/tests/functional/adapters/test_l7policy_adapter.py
+++ b/f5_openstack_agent/tests/functional/adapters/test_l7policy_adapter.py
@@ -106,7 +106,6 @@ def test_adapter_redirect_to_pool_file_type_beginswith(
                 'actions': [
                     {
                         'forward': True, 'name': 0, 'request': True,
-                        'name': '0',
                         'pool': u'/Project_test/Project_test_pool'
                     }
                 ],
@@ -413,7 +412,7 @@ def test_adapter_redirect_to_pool_file_type_not_beginswith(
                 'actions': [
                     {
                         'forward': True, 'name': 0, 'request': True,
-                        'name': '0', 'pool': u'/Project_test/Project_test_pool'
+                        'pool': u'/Project_test/Project_test_pool'
                     }
                 ],
                 'conditions': [

